internal/util/utiltest/mockcopiable.go

```diff
@@ -2,14 +2,17 @@ package utiltest
 
 import "github.com/Vungle/vungo/internal/util"
 
+// MockCopiable mock file for testing copiable structs
```

Comment:
please add period in the end of the comment.

Reply:
Add period to the end of one comment. Check other files and append a period too.